docs/pages/operations-guide.md

```diff
@@ -151,47 +177,53 @@ Plans have a fixed three-level hierarchy. Plans contain Phases, and Phases conta
         - Data Node 1 Step
         - Data Node 2 Step
 
-As you can see, in addition to the default Deployment and Recovery plans, this Scheduler also has auxiliary Plans which support custom behavior, specifically one Plan which handles backing up Index nodes, and another for backing up Data nodes. In practice there would likely also be Plans for restoring these backups but those are omitted here for brevity.
+As you can see, in addition to the default Deployment and Recovery plans, this Scheduler also has auxiliary Plans that support custom behavior, specifically one Plan that handles backing up Index nodes, and another for that backs up Data nodes. In practice, there would likely also be Plans for restoring these backups. Those are omitted here for brevity.
+
+In short, Plans are the SDK's abstraction for a sequence of tasks to be performed by the Scheduler. By default, these include deploying and maintaining the cluster, but additional maintenance operations may also be fit into this structure.
+
+### Recovery Plan <!-- I'd like someone to look this over -->
 
-So in short, Plans are the SDK's abstraction for a sequence of tasks to be performed by the Scheduler. By default these include deploying and maintaining the cluster, but additional maintenance operations may also be fit into this structure.
+Another common/default Plan is the Recovery Plan, which handles bringing back failed tasks. The Recovery Plan listens for offers that can be used to bring back those tasks and then relaunches tasks against those offers.
 
-### Recovery Plan
+The Scheduler learns whether a task has failed by receiving Task Status updates from Mesos. Task Status updates can be sent during startup to let the scheduler know when a task has started running, to know when the task has exited successfully, or to know when the cluster has lost contact with the machine hosting that task.
```

Comment:
could remove 'during startup': "Task Status updates can be sent to let the Scheduler know"